code/game/machinery/constructable_frame.dm

```diff
@@ -230,7 +230,7 @@
 							new_machine.circuit.moveToNullspace()
 							QDEL_NULL(new_machine.circuit)
 						for(var/obj/old_part in new_machine.component_parts)
-							// Move to nullspace & delete other parts like cells, beakers etc.
+							// Move to nullspace & delete
```

Comment:
I think the other part of the comment was fine since it makes it clear that this isn't typed by mistake. If you aren't going to keep it remove this comment entirely because the comment is useless

Reply:
I'll delete it then or its going to be a long comment listing every part obj part

Reply:
That's what "etc" is for :P